rust_src/build.rs

```diff
@@ -110,19 +110,29 @@ where
             },
 
             ParseState::LispFn(name) => {
-                if line.starts_with("pub") || line.starts_with("fn") {
-                    if let Some(func) = name.or_else(|| get_function_name(&line)) {
-                        write!(out_file, "pub use {}::F{};\n", modname, func)?;
-                        exported.push(func);
-                    }
+                let name = name.or_else(|| get_function_name(&line)).unwrap();
+
+                if line.starts_with("fn") {
+                    eprintln!(
+                        "
+`{}` is not public in the {} module at line {}.
+lisp_fn functions are meant to be used from Rust as well as in lisp code.",
+                        name,
+                        modname,
+                        lineno,
+                    );
+                    process::exit(1);
```

Comment:
Why use panic in some cases and process::exit in others?

Reply:
The panics are exceptions. The exits are user errors.